services/sources/gutenberg_service.py

```diff
@@ -40,7 +40,8 @@ def get_records(
         self,
         start_timestamp: Optional[datetime]=None,
         offset: int=0,
-        limit: Optional[int]=None
+        limit: Optional[int]=None,
+        record_only: bool=False,
```

Comment:
this is kind of hacky but the gutenbergmapping class is not pickleable with multiprocessing.

Reply:
I like that. I think the goal with the eventual ETL pipeline is that we don't return mapping objects in the future!